src/styles/main.scss

```diff
@@ -1,84 +1,104 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  min-height: 100vh;
-  background: #fbf8ef;
-  font-family: sans-serif;
-  font-size: 24px;
-  font-weight: 900;
+  padding: 0;
+}
+
+.page {
+  font-size: 7.5vmin;
+  font-family: Arial, Helvetica, sans-serif;
+
+  &__body {
+    margin: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-height: 100vh;
+    background-color: #333;
+    font-family: sans-serif;
+    font-size: 24px;
+    font-weight: 900;
+  }
 }
 
 .field-cell {
-  background: #d6cdc4;
-  width: 75px;
-  height: 75px;
-  border-radius: 5px;
-  color: #776e65;
+  background-color: #444;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)